server/tso/global_allocator.go

```diff
@@ -157,17 +157,17 @@ func (gta *GlobalTSOAllocator) syncMaxTS(ctx context.Context, dcLocationMap map[
 	maxRetryCount := 1
 	for i := 0; i < maxRetryCount; i++ {
 		// Collect all allocator leaders' client URLs
-		allocatorLeaders, err := gta.allocatorManager.GetLocalAllocatorLeaders()
+		allocatorLeaders, err := gta.allocatorManager.GetLocalAllocatorLeadersMember()
```

Comment:
`GetLocalAllocatorLeadersMember` name is misleading.

I prefer `GetAllLocalAllocator`